### PR TITLE
Add AUR integration.

### PR DESCRIPTION
**Pull request by JP-Ellis**
Add badge for the [Arch User Repository (AUR)](https://aur.archlinux.org).  Includes badges to
display the current version, number of votes, and license.

Unfortunately, their [API](https://aur.archlinux.org/rpc.php) does not display downloads at this stage.

### CODE REVIEW

**Comment by JP-Ellis** [comment]
Is there anything that should be changed before this gets merged?

**Comment by JP-Ellis** [comment]
I have rebased this pull request so that it can be merged.

Anything needing to be changed before it gets merged?

**Comment by kyrias** [comment]
Any real reason for using v/o/l instead of version/votes/license?

**Comment by kyrias** [comment]
It just seems to make the urls more confusing to me.

**Comment by JP-Ellis** [comment]
No reason, I just remember looking at another one and seeing abbreviations down to 1 letter.  I'll amend the pull request to be more verbose.

**Comment by JP-Ellis** [comment]
Is there anything more you need before this gets merged?  Or is there a reason to not merge this pull request?

**Comment by JP-Ellis** [comment]
@espadrine Do you need anything more for this pull request to be merged?  Or do you not intend to merge it?

**Comment by JP-Ellis** [comment]
@espadrine Over two months and no feedback on this pull request.  Should I close it?

File: server.js [hunk 1]
```diff
@@ -4734,6 +4734,52 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// Arch user repository (AUR) integration.
+camp.route(/^\/aur\/(version|votes|license)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+    var info = match[1];
+    var package = match[2];
```
Comment (espadrine) [inline server.js]: `package` is a reserved word in JS. Could you use `pkg` instead, please?

File: server.js [hunk 2]
```diff
@@ -4734,6 +4734,52 @@ cache(function(data, match, sendBadge, request) {
...
+                badgeData.text[1] = vdata.version;
+                if (data.OutOfDate === null) {
+                    badgeData.colorscheme = 'blue';
+                } else {
+                    badgeData.colorscheme = 'orange';
+                }
+                sendBadge(format, badgeData);
+            } else if (info === 'votes') {
+                var votes = data.NumVotes;
+                badgeData.text[1] = votes;
```
Comment (espadrine) [inline server.js]: Do you want to have the left-hand side part of the badge say "votes", to be descriptive?

File: server.js [hunk 3]
```diff
@@ -4734,6 +4734,52 @@ cache(function(data, match, sendBadge, request) {
...
+                badgeData.colorscheme = floorCountColor(votes, 2, 20, 60);
+                sendBadge(format, badgeData);
+            } else if (info === 'license') {
+                var license = data.License;
+                badgeData.text[0] = "license";
+                badgeData.text[1] = license;
+                badgeData.colorscheme = 'blue';
+                sendBadge(format, badgeData);
```
Comment (espadrine) [inline server.js]: Could you put all the `sendBadges()` outside, after the series of if / else clauses, please?

File: server.js [hunk 4]
```diff
@@ -4734,6 +4734,52 @@ cache(function(data, match, sendBadge, request) {
...
+            } else {
+                throw 0;
```
Comment (espadrine) [inline server.js]: Could you remove that clause, please?

File: server.js [hunk 5]
```diff
@@ -4734,6 +4734,52 @@ cache(function(data, match, sendBadge, request) {
...
+            }
+        } catch(e) {
+            console.log(e);
```
Comment (espadrine) [inline server.js]: Could you remove the log, please?

**Comment by espadrine** [comment]
Thank you for contributing!
I understand that you were disappointed with the speed at which I reviewed this patch. For that, I am sorry. I'll try to make the remainder of the review faster.

Could you please change the indentation to 2-space, please?

**Comment by JP-Ellis** [comment]
I have updated the pull request as per your pull-request.  Thanks for taking a look and sorry for all the little mistakes.

**Comment by espadrine** [comment]
Thanks for contributing. Your changes are live.

**Pull request merged**